common/src/main/java/com/wynntils/models/items/properties/UnidentifiedItemProperty.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.properties;
+
+/**
+ * Represents an item that may or may not be identified at an item identifier.
+ */
+public interface UnidentifiedItemProperty {
```

Comment:
I'm not sure I like these WynnItem hierarchy changes. There's something ... itchy ... about them, that I can't really put my finger to, without thinking a bit more. So let me start by turning the tables: Why do you think you need these changes, instead of just checking for GearBoxItem vs GearItem.isUnidentified()?

Reply:
I think you mean that having "UnidentifedItemProperty" return false for isUnidentified is really weird.